source/includes/administration/_organization_discounts.md

```diff
@@ -0,0 +1,291 @@
+`durationDays`<br/>*integer* | Duration of the discount once it has been applied to a customer. If not provided the discount will last indefinitely, or until credit values are reached.
+`type`<br/>*enum* | The type of the discount. It could be either "PERCENTAGE" or "CREDIT".
+`packageDiscount`<br/>*number* | The discount value that will be applied to all products within the applied pricing.
+`scope`<br/>*enum* | The scope of the discount. It could be either "ALL_PRODUCTS", "CATEGORIES" or "PRODUCTS".
+`discountedCategories`<br/>*object[string, number]* | A mapping between category IDs and discount values. All pricing products within a category will have the discount value applied to them.
+`name`<br/>*object[string, string]* | The name translations of the discount.
+`startDate`<br/>*date* | The start date of the discount.
+`status`<br/>*enum* | The status of the discount. Possible values are: UPCOMING, CURRENT, ENDED.
+
+<!-------------------- CREATE ORGANIZATION DISCOUNT -------------------->
+### Create organization discount
+
+`POST /organizations/:organization_id/discounts`
+
+Create a new discount
+
+```shell
+# Creates a new discount
+curl -X POST "https://cloudmc_endpoint/rest/organizations/efd32752-c6f2-45cf-b494-cc6be8a45845/discounts" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+> Request body example:
+
+```json
+{
+  "name": {
+    "en": "Summer Discount",
+    "fr": "Réduction Estival"
+  },
+  "startDate": "2021-07-23T00:00:00.000Z",
+  "type": "PERCENTAGE",
+  "durationDays": 60,
+  "discountedProducts": {},
+  "discountedCategories": {
+    "8cf73cc0-b86e-49b4-a102-6102894f7955": 2, 
+    "00358632-5c9a-4164-a9a9-df271a9c06a9": 22
+  },
+  "scope" "CATEGORIES", 
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": {
+      "discountedProducts": {},
+      "durationDays": 60,
+      "type": "PERCENTAGE",
+      "scope": "CATEGORIES",
+      "discountedCategories": {
+        "8cf73cc0-b86e-49b4-a102-6102894f7955": 2, 
+        "00358632-5c9a-4164-a9a9-df271a9c06a9": 22
+      },  
+      "name": {
+        "en": "Summer Discount",
+        "fr": "Réduction Estival"
+      },
+      "id": "18db7bc6-8be1-48bb-bab1-77a7d696fa3b",
+      "startDate": "2021-07-23T00:00:00.000Z"
+  }
+}
+```
+
+Required | &nbsp;
+------- | -----------
+`name`<br/>*object[string, string]* | The name translations of the discount.
+`type`<br/>*enum* | The type of the discount. It can be either "PERCENTAGE" or "CREDIT".
+`startDate`<br/>*date* | The start date of the discount.
+`scope`<br/>*enum* | The scope of the discount. It can be either "ALL_PRODUCTS", "CATEGORIES" or "PRODUCTS".
+`packageDiscount`<br/>*number* | The discount value that will be applied to all products within the applied pricing. Only required if the scope is "ALL_PRODUCTS". The value must be between (0,100] for a percentage discount and greater than 0 for a credit.
```

Comment:
If this is provided alongside the `discountedCategories` or `discountedProducts`, how does the API react?

Reply:
it's identical to the package discount, meaning it depends on the `scope`, the fields are essentially ignored

basically something like this

```
if scope == ALL_PRODUCTS
  set categories = []
  set products = []
else if scope == CATEGORIES
  set packageDiscount = null
  set products = []
else if scope == PRODUCTS
  set packageDiscount = null
  set categories = []
```

I don't know why the scope isn't computed from the fields 🤷

Reply:
Thanks!